CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v1.13.3 (2023-04-17)
+
+## OS Changes
+
+* Update kernel-5.10 to 5.10.173 and kernel-5.15 to 5.15.102 ([#2948], [#3002])
+* Backport ip6tables v1.8.9 rule check fix ([#3001])
```

Comment:
```suggestion
* Fix check for rule existence in ip6tables v1.8.9  ([#3001])
```

https://github.com/bottlerocket-os/bottlerocket/pull/2976 is not a backport but a custom patch to fix the issue.